docs/keto/guides/migrating-legacy-policies.mdx

```diff
@@ -0,0 +1,112 @@
+---
+id: migrating-legacy-policies
+title: Migrating policies from 0.5 to 0.6
+---
+
+0.6 release makes Ory Access Control Policy DSL modeled after AWS IAM Policies obsolete. This guide will help you to rewrite your policies in to [relation-tuples](https://www.ory.sh/docs/keto/concepts/relation-tuples). You can read [The Evolution of Ory Keto: A Global Scale Authorization System](https://www.ory.sh/keto-zanzibar-evolution/) blogpost to understand a benefits of 0.6 release
+
+## Legacy rules example
+
+The policy below allows `Alice` and `Bob` to create/read/modify/delete `blog_posts:my-first-blog-post`, `blog_posts:2`, and `blog_posts:3`.
+
+```json
+{
+  "subjects": ["alice", "bob"],
+  "resources": [
+    "blog_posts:my-first-blog-post",
+    "blog_posts:2",
+    "blog_posts:3"
+  ],
+  "actions": ["delete", "create", "read", "modify"],
+  "effect": "allow"
+}
+```
+
+## Rewriting it to relation tuples
+
+According to the example above we need to create required [namespace](https://www.ory.sh/docs/keto/concepts/namespaces) and [relationship tuple](https://www.ory.sh/docs/keto/concepts/relation-tuples)
```

Comment:
Just a tiny naming thing, but we use relation tuple everywhere, not relationship.
```suggestion
According to the example above we need to create required [namespace](https://www.ory.sh/docs/keto/concepts/namespaces) and [relation tuple](https://www.ory.sh/docs/keto/concepts/relation-tuples)
```